src/Microsoft.Data.Sqlite/project.json

```diff
@@ -1,6 +1,13 @@
 {
   "version": "1.0.0-*",
-  "description": "SQLite implementation of the System.Data.Common provider model.",
+  "description": "Provides a data provider for SQLite based on the System.Data.Common contract.\r\n\r\nCommonly Used Types:\r\nMicrosoft.Data.Sqlite.SqliteCommand\r\nMicrosoft.Data.Sqlite.SqliteConnection\r\nMicrosoft.Data.Sqlite.SqliteConnectionStringBuilder\r\nMicrosoft.Data.Sqlite.SqliteDataReader\r\nMicrosoft.Data.Sqlite.SqliteException\r\nMicrosoft.Data.Sqlite.SqliteFactory\r\nMicrosoft.Data.Sqlite.SqliteParameter\r\nMicrosoft.Data.Sqlite.SqliteTransaction",
```

Comment:
Hmmm, perhaps we could use something like...

> Provides a provider that provides SQLite data access provided you are ok with not being provided with typed data set and other provider things not provided by System.Data.Common

...on a serious note, my vote is to keep the old wording and just add the "Commonly used types" part